eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/service/ClusteredVariantsBeaconService.java

```diff
@@ -0,0 +1,89 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.ws.service;
+
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.ampt2d.commons.accession.core.models.AccessionWrapper;
+import uk.ac.ebi.ampt2d.commons.accession.hashing.SHA1HashingFunction;
+import uk.ac.ebi.ampt2d.commons.accession.rest.dto.AccessionResponseDTO;
+
+import uk.ac.ebi.eva.accession.core.ClusteredVariant;
+import uk.ac.ebi.eva.accession.core.ClusteredVariantAccessioningService;
+import uk.ac.ebi.eva.accession.core.IClusteredVariant;
+import uk.ac.ebi.eva.accession.core.summary.ClusteredVariantSummaryFunction;
+import uk.ac.ebi.eva.commons.beacon.models.BeaconAlleleRequest;
+import uk.ac.ebi.eva.commons.beacon.models.BeaconAlleleResponse;
+import uk.ac.ebi.eva.commons.beacon.models.BeaconDatasetAlleleResponse;
+import uk.ac.ebi.eva.commons.beacon.models.Chromosome;
+import uk.ac.ebi.eva.commons.core.models.VariantType;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+
+@Service
+public class ClusteredVariantsBeaconService {
+
+    private ClusteredVariantAccessioningService clusteredVariantService;
+
+    private Function<IClusteredVariant, String> hashingFunctionClusteredVariants =
+            new ClusteredVariantSummaryFunction().andThen(new SHA1HashingFunction());
```

Comment:
I know that maybe we already redefine the summary function, but I'd prefer to have as few definitions as possible. see if you can use `AccessioningService::get(List<Model>)` because then this is not needed here.